pep-0690.rst

```diff
@@ -193,6 +193,17 @@ a list of module names within which all imports will be eager::
 The effect of this is also shallow: all imports within ``one.mod`` will be
 eager, but not imports in all modules imported by ``one.mod``.
 
+``set_eager_imports()`` can instead take a callback which receives a module name and returns
+whether the import of this module should be eager::
```

Comment:
It's unclear what "instead" is refering to in this sentence. Perhaps just set out what the function does, and then how it is different to (the other thing that 'instead' is refering to)?

A

Reply:
I think it makes sense with the fuller context of the previous paragraphs? The immediately preceding text describes (and shows) how `set_eager_imports` works when it takes a list of module names. Then this proceeds to describe how it can instead take a callback. So I think the text already does what you recommend:

> Perhaps just set out what the function does, and then how it is different